scalableminds/webknossos
# Fix deprecation warnings for antd <Collapse> components

Fix deprecation warnings for antd `<Collapse>` components. API changed to `items` props instead of React children/`<Panel>`.

See https://ant.design/components/collapse#collapsepanel

### URL of deployed dev instance (used for testing):
- https://___.webknossos.xyz

### Steps to test:
- Navaigate to any view using a `<Collapse />`
- Open web dev console
- There should not be any deprecation warnings

### TODOs:
- [x] Fix typescript errors

### Issues:
- Follow up to #7522

------
(Please delete unneeded items, merge only when none are left open)
